### PR TITLE
Create monthly-february-2020.md

### PR DESCRIPTION
**Pull request by WT-Liu**
### What is changed?
monthly blog update for Feb. 2020

### Any related PRs or issues?
N/A

### Which version does your change affect?
N/A

### CODE REVIEW

File: content/blog/monthly-february-2020.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# This Month in TiKV - February 2020
+
+TiKV authors
+
+March 12, 2020
+
```
Comment (dcalvin) [inline content/blog/monthly-february-2020.md]: These are metadada. Please look at existing files on how to write them @WT-Liu

**Comment by nrc** [comment]
You need to sign-off GitHub's automatic commits yourself manually.

File: content/blog/monthly-february-2020.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+As the cold winter has passed, we are welcoming a beautiful spring with blossoms, gentle breeze, and of course our monthly TiKV update, covering February 2020.
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]: I love it!

File: content/blog/monthly-february-2020.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Let’s get started!
+
+### Releases
+
+This month our team made 2 TiKV releases (3.0.10 and 4.0.0- beta.1). The 3.0.10 release is a minor one with bug fixes and the 4.0.0- beta.1 release includes compatibility improvements, 2 new features, and bug fixes.
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]:
```suggestion
This month our team made 2 TiKV releases (3.0.10 and 4.0.0-beta.1). The 3.0.10 release has only bug fixes; the 4.0.0-beta.1 release includes compatibility improvements, two new features, and bug fixes. 
```

What are the new features?

File: content/blog/monthly-february-2020.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+
+### Notable PRs
+
+*   [@BusyJay](https://github.com/BusyJay) upgraded thread pool to support yield in [#6487](https://github.com/tikv/tikv/pull/6487), making tasks handled more fairly.
+*   [@kennytm](https://github.com/kennytm) added backup data support to S3 storage on 3.1 version in[ #6536](https://github.com/tikv/tikv/pull/6536).
+*   [@BusyJay](https://github.com/BusyJay) introduced pre-transfer leader to make raft leadership transfer more smoothly in [#6539](https://github.com/tikv/tikv/pull/6539).
+*   [@5kbpers](https://github.com/5kbpers) added command observer to observe executed command in the apply thread pool in[ #6602](https://github.com/tikv/tikv/pull/6602). This is the preparation for CDC (capture data changes).
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]:
```suggestion
*   [@5kbpers](https://github.com/5kbpers) added command observer to observe executed command in the apply thread pool in[ #6602](https://github.com/tikv/tikv/pull/6602). This is in preparation for CDC (capture data changes).
```

File: content/blog/monthly-february-2020.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+*   [@sticnarf](https://github.com/sticnarf) used the unified thread pool in [#6593](https://github.com/tikv/tikv/pull/6593) to handle point get requests and coprocessor requests.
+
+
+*   [@nolouch](https://github.com/nolouch) verified online reload new [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) certificate in [#2162](https://github.com/pingcap/pd/pull/2162).
+*   [@Luffbee](https://github.com/Luffbee) introduced a solution-based balance solver in [#2141](https://github.com/pingcap/pd/pull/2141).
+
+*   [@iosmanthus](https://github.com/iosmanthus) upgraded DAG framework in [#5871](https://github.com/tikv/tikv/pull/5817), making TiKV never fallback to the volcano model mode.
+*   [@TennyZhuang](https://github.com/TennyZhuang) supported new index encoding with collation support in [#6685](https://github.com/tikv/tikv/pull/6685).
+*   [@youjiali1995](https://github.com/youjiali1995) made it possible to return values when pessimistic lock successfully to reduce RPC in [#6696](https://github.com/tikv/tikv/pull/6696).
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]:
```suggestion
*   [@youjiali1995](https://github.com/youjiali1995) made it possible to return values with successful pessimistic lock responses to reduce network traffic in [#6696](https://github.com/tikv/tikv/pull/6696).
```

File: content/blog/monthly-february-2020.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+**Call for participation**
+
+[@overvenus](https://github.com/overvenus) opened issue [#6734](https://github.com/tikv/tikv/issues/6734), requesting to update PD member information every 10 minutes in order to minimize the time window of holding stale information.
+
+[@hicqu](https://github.com/hicqu) suggested in [#6666](https://github.com/tikv/tikv/issues/6666) that  we  should remove `shared_block_cache` from `engine_trait::KvEngines` to simplify `KvEngine::flush_metrics` and he clarified 2 reasons for this suggestion. We have seen one vote up. What’s your opinion?
+
+[@yiwu-arbug](https://github.com/yiwu-arbug) created [#6506](https://github.com/tikv/tikv/issues/6506) and [#6507](https://github.com/tikv/tikv/issues/6507) and [@Little-Wallace](https://github.com/Little-Wallace) created [#6508](https://github.com/tikv/tikv/issues/6508) to ask if some students would be interested in participating in [GSoC](https://summerofcode.withgoogle.com/) by working on these projects. Follow these issues if you are interested.
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]:
```suggestion
[@yiwu-arbug](https://github.com/yiwu-arbug) and [@Little-Wallace](https://github.com/Little-Wallace) created [#6506](https://github.com/tikv/tikv/issues/6506), [#6507](https://github.com/tikv/tikv/issues/6507), and [#6508](https://github.com/tikv/tikv/issues/6508) to ask if students would be interested in participating in [GSoC](https://summerofcode.withgoogle.com/) by working on these projects. Follow these issues if you are interested.
```

File: content/blog/monthly-february-2020.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+For more detailed and comprehensive information about TiDB and TiKV, we have weekly updates. The following cover February.
+
+[January 20 ~ February 02, 2020](https://pingcap.com/weekly/2020-02-03-tidb-weekly/)
+
+[February 03 ~ February 09, 2020](https://pingcap.com/weekly/2020-02-10-tidb-weekly/)
+
+[February 10 ~ February 16, 2020](https://pingcap.com/weekly/2020-02-17-tidb-weekly/)
+
+[February 17 ~ February 23, 2020](https://pingcap.com/weekly/2020-02-24-tidb-weekly/)
```
Comment (nrc) [inline content/blog/monthly-february-2020.md]: Can you make these a bulleted list please?

File: content/blog/monthly-february-2020.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
+---
+title: This Month in TiKV - February 2020
+date: 2020-03-12
+author: TiKV authors
```
Comment (dcalvin) [inline content/blog/monthly-february-2020.md]:
```suggestion
author: TiKV Authors
```

**Pull request merged**